package.json

```diff
@@ -41,7 +41,7 @@
         "test:all": "wsrun --fast-exit --serial --exclude-missing -p $PKG -c test",
         "test:contracts": "wsrun --serial -p $(echo ${npm_package_config_contractsPackages} ${npm_package_config_ignoreTestsForPackages} | tr ' ' '\n' | sort | uniq -u | tr '\n' ' ') --fast-exit --exclude-missing -c test",
         "test:contracts:all": "wsrun --serial -p ${npm_package_config_contractsPackages} --fast-exit --exclude-missing -c test",
-        "test:links": "yarn check-md",
+        "test:links": "yarn check-md --fix",
```

Comment:
This is run in CI and should not be writing there. If something is broken it should fail build not fix and fudge it.